README.md

```diff
@@ -85,7 +81,7 @@ They exist to document what may be implemented by [Nostr](https://github.com/nos
 | `6`     | Repost                     | [18](18.md) |
 | `7`     | Reaction                   | [25](25.md) |
 | `8`     | Badge Award                | [58](58.md) |
-| `16`    | Generic Repost             | [18](18.md) |
+| `18`    | Generic Repost             | [18](18.md) |
```

Comment:
Are we changing the event kind of Generic Repost to 18? Looks like a simple error in the PR.

Reply:
Oops, I got confused thinking this was the NIP number, not the kind number, will fix.